packages/protocol-utils/src/orders.ts

```diff
@@ -129,6 +135,35 @@ export abstract class OrderBase {
         }
     }
 
+    /*
+     * This is an advanced order signature function that should only be used if you know what you're doing.
+     * It allows you to bring your own wallet signature callback function, this can be used to sign orders
+     * with a wallet that is otherwise unsupported by Web3Wrapper.
+     * The signature function needs to implement correct signing of the data and return the results as a string.
+     */
+    public async _getSignatureWithSignFunctionAsync(
```

Comment:
why _ prefix?

Reply:
to deter people from using this function 😂